### PR TITLE
Update the map command output

### PR DESCRIPTION
**Pull request by yuce**

### CODE REVIEW

File: internal/it/util.go [hunk 1]
```diff
@@ -61,6 +61,7 @@ var defaultDedicatedTestCluster = NewSingletonTestCluster(defaultDedicatedCluste
 	port := NextPort()
 	return rc.startNewCluster(MemberCount(), XMLConfig(defaultDedicatedClusterName, port), port)
 })
+var DefaultDedicatedTestCluster = defaultDedicatedTestCluster
```
Comment (kutluhanmetin) [inline internal/it/util.go:64]: I think this change is not related to this PR

**Review by kutluhanmetin — APPROVED** [review]
just a minor command

File: internal/it/util.go (hunk 1 above)
Comment (yuce) [reply internal/it/util.go]: Removed at: https://github.com/hazelcast/hazelcast-commandline-client/pull/374/commits/8d957cf473ec7d3b64ffb964f223c176feea2d3c

**Review by kutluhanmetin — APPROVED** [review]
